src/models/UserMapper.php

```diff
@@ -544,6 +544,31 @@ public function getUserIdFromUsername($username)
     }
 
     /**
+     * Function to get just the user ID
+     *
+     * @param string $fullname The users full name we're looking for
+     *
+     * @return $user_id The user's ID (or false, if we didn't find her)
+     */
+    public function getUserIdFromFullname($fullname)
```

Comment:
We don't ever have 2 users with the same name. Ever! 

Damn! Usually that's my part to spot it. No idea what what happened there. I'll check that back with the legacy site as I was under the impression that you could add a user with it's full name there and didn't further think about it. I'll probably change it to the users login-name.

Thanks for spotting!!